lib/report_email_sender.rb

```diff
@@ -0,0 +1,41 @@
+class ReportEmailSender
```

Comment:
yeah no idea rereading that -> https://devcenter.heroku.com/articles/sendgrid#ruby-rails <- page why I went for the gem and not actionmailer